docs/migration-v8.md

```diff
@@ -19,6 +19,16 @@ Welcome to the migration guide for Polly's v8 release. Version 8 of Polly brings
 > [!NOTE]
 > Please read the comments in the code carefully for additional context and explanations.
 
+## Polly or Polly.Core package
+
+When you do your migration process it is recommended to follow these steps:
+
+- Upgrade the `Polly` package version from 7.x to 8.x
+  - Your previous policies should run smoothly without any change
+- Migrate your V7 policies to V8 strategies one-by-one gradually
+  - Test your migrated code in a QA environment before publishing your changes into production
```

Comment:
It really depends how the unit tests were written. I have seen many unit tests which were relying on Polly's interfaces. Those tests can't be used after the migration.

When I say  _test your migrated code in a QA environment_ that implies to me the verification of integration and regression as well.

Did I convince you?

Reply:
My point is more about _"shift left"_ - I think we should mention tests that happen before deployment. If a person doesn't have any, or they don't actually test Polly, then they'll just skip over it anyway. Otherwise we should be less specific and just say "test your code" and leave it up to the reader to interpret what that means for them.